packages/components/src/tools-panel/test/index.tsx

```diff
@@ -1126,9 +1160,13 @@ describe( 'ToolsPanel', () => {
 			expect( announcement ).toHaveAttribute( 'aria-live', 'assertive' );
 
 			const disabledResetAllItem = await screen.findByRole( 'menuitem', {
-				disabled: true,
```

Comment:
I was curious why we preferred to have a separate assertion for this when we can just have it in the query?

Reply:
That's because apparently the `byRole` selector doesn't accept the `disabled` option ([docs](https://testing-library.com/docs/queries/byrole/#api)) — which means that effectively this test was not testing what it was supposed to!

Reply:
Oh, right! Great catch!

Reply:
You can thank TypeScript :D